### DISCUSSION TITLE
Expression mutability annotation

### DISCUSSION THREAD

**Discussion by ethanrjs** [SkriptLang/Skript#4925] [Suggestions] [answered]
Add an annotation for expressions saying whether or not they are settable/mutable values. i.e, `the lines of a sign` are mutable but the `players name` is not. 

I have repeatedly wasted time wondering whether or not I can change the value of an expression, and the documentation does not clarify whether or not I can. It would be a very nice time saving QoL feature :)

**Comment by AyhamAl-Ali** [comment, marked as answer]
Good suggestion 👍 it has already been suggested here #4157 and being worked on so expect to see it soon \:D
